src/orchestratord/src/controller/materialize/console.rs

```diff
@@ -156,7 +187,8 @@ fn create_console_deployment_object(
         ..Default::default()
     }];
 
-    if config.enable_tls {
+    if mz.spec.console_external_certificate_spec.is_some() {
+        // TODO define volumes, volume_mounts, and any arg changes
```

Comment:
do you want to wire in the console certs in this PR? or leave for a follow up

Reply:
I was going to leave it for a follow up. I don't know how the console works, and didn't want to block this on me figuring that out.